docker/Makefile

```diff
@@ -6,7 +6,11 @@ ALPINE_REGISTRY		:= docker.pkg.github.com/v6d-io
 ALPINE_IMAGE		:= v6d/vineyardd
 ALPINE_TAG			:= alpine-latest
 
-all: builder docker-build docker-push
+PYTHON_DEV_REGISTRY	:= docker.pkg.github.com/v6d-io
+PYTHON_DEV_IMAGE	:= v6d/vineyard-python-dev
+PYTHON_DEV_TAG		:= latest
+
+all: builder docker-build build-python-dev docker-push
```

Comment:
The `build-python-dev` shouldn't be included in `all`.

Reply:
Done